src/datastructure/rollback_unionfind.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include <bits/stdc++.h>
+using namespace std;
+
+struct RollbackUnionFind {
+    vector<int> data;
+    stack<pair<int, int>> history;
+    int inner_snap = 0;
+    RollbackUnionFind(int n) { data.resize(n, -1); }
+
+    bool unite(int x, int y) {
+        x = find(x);
+        y = find(y);
```

Comment:
```suggestion
        x = find(x), y = find(y);
```

簡単な代入なので、カンマで続けて行を節約していいと思います！